### DISCUSSION TITLE
Axis-angle misorientation map

### DISCUSSION THREAD

**Discussion by Rhalder322** [mtex-toolbox/mtex#2200] [Feature Requests]
Is it possible to generate the axis-angle misorientation map for a given reconstructed beta map for Ti64 using MTEX/MTEX ORTools?

**Comment by ralfHielscher** [comment]
Do you mean the misorientation between the reconstructed beta phase and the measured phase?
Ralf.

**Reply by Rhalder322** [reply]
I meant to say "axis-angle misorientation between reconstructed beta grains".
